<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Plain Text Edit</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>120</y>
     <width>191</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btn_printPTE">
      <property name="text">
       <string>Print Plain Text Edit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_clearPTE">
      <property name="text">
       <string>Clear Plain Text Edit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_appendPT">
      <property name="text">
       <string>Append Plain Text</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_Test">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>361</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
